modules/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -18,7 +18,7 @@ import (
 // in client state that must be the same across all valid Tendermint clients for the new chain.
```

Comment:
maybe add a comment that zeroing out the submitted client prevents the proposal from containing information governance is not actually voting on, see previous CHANGELOG comment